ocaml/networkd/bin/network_server.ml

```diff
@@ -269,6 +349,22 @@ module Interface = struct
   let get_all dbg () =
     Debug.with_thread_associated dbg (fun () -> Sysfs.list ()) ()
 
+  let get_interface_positions dbg () =
+    Debug.with_thread_associated dbg
+      (fun () ->
+        if Network_utils.device_already_renamed then
```

Comment:
Would the `config.interface_order` (either `Some _`, or `None`) stand for this as well?